src/components/common/AsideNavBar/AsideNavBar.jsx

```diff
@@ -0,0 +1,123 @@
+                        </NavLink>
+                    </li>
+                    <li>
+                        <NavLink style={getActiveStyle}
+                            to={ROUTES.ROUTE_PATH_EXPLOREPAGE}>
+                            <IconsExploreNavbar height="1.5em" width="1.5em" />
+                            <p className="title-hide-responsive">Explore</p>
+                        </NavLink>
+                    </li>
+                    <li>
+                        <NavLink style={getActiveStyle}
+                            to={ROUTES.ROUTE_PATH_BOOKMARKPAGE}>
+                            <IconsBookmark height="1.5em" width="1.5em" />
+                            <p className="title-hide-responsive">Bookmark</p>
+                        </NavLink>
+                    </li>
+                    <li>
+                        <NavLink style={getActiveStyle}
+                            to={ROUTES.ROUTE_PATH_ProfilePage}>
+                            <IconsProfile height="1.5em" width="1.5em" />
+                            <p className="title-hide-responsive">Profile</p>
+                        </NavLink>
+                    </li>
+                    <li>
+                        
+                        {!userState.token ? (
+                            <NavLink style={getActiveStyle}
+                            to={ROUTES.ROUTE_PATH_LoginPage}>
+                            <IconsProfile height="1.5em" width="1.5em" />
+                            <p className="title-hide-responsive">Login</p>
+                        </NavLink>
+                            ) : (
+                            <button
+                                className="btn primary-outline-btn-md"
+                                onClick={LogoutHandler}
+                            >
+                                Logout
+                            </button>
+                            )}
+                    </li>
+                    {/* {
+                        !auth.user ?
+
+                            <span onClick={
+                                () => {
+                                    console.log("login");
+                                    setmodalToggle(true);
+                                }} className='create-btn' style={{ width: "100%" }}>
+                                <p className="title-hide-responsive">Login</p>
+                                <BiBoxArrowInLeft height="1.5em" width="1.5em" />
+                            </span>
+
+                            : <li>
+                                <NavLink style={getActiveStyle}
+                                    to={ROUTE_PATH_PROFILEPAGE}>
+                                    <BiPersonHearts height="1.5em" width="1.5em" />
+                                    <p className="title-hide-responsive">Profile</p>
+                                </NavLink>
+                            </li>
```

Comment:
remove comments

Reply:
this is comment because the logic for auth is still not fully done so the code is comment temporary i need it for showing username